packages/components/src/color-palette/test/index.tsx

```diff
@@ -201,4 +212,41 @@ describe( 'ColorPalette', () => {
 			screen.getByRole( 'button', { name: 'Clear' } )
 		).toBeInTheDocument();
 	} );
+
+	it( 'should display the selected color name and value', async () => {
+		const user = userEvent.setup();
+
+		const { container } = render( <UnmockedColorPalette /> );
+
+		expect( screen.getByText( 'No color selected' ) ).toBeVisible();
+		expect(
+			// Relying on implementation details here, but it's the only way I could come up with.
+			// eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+			container.querySelector(
+				'.components-color-palette__custom-color-value'
+			)
+		).toHaveTextContent( '' );
```

Comment:
I'd almost remove this check entirely for now, it is so specific and implementation details-oriented that it doesn't really bring much value to the test. What do you think?

Reply:
Yeah, I definitely questioned it too. On the one hand, including a test is the only we we can ensure a future regression doesn't cause a stale or incorrect value from showing up when it shouldn't. On the other this is a pretty fragile test.

In the end, we'll still be testing the color name, and both of these values flow from the component's `value` prop, so as long as the one is behaving as expected we can be _partially_ confident in the other. I'll remove the check for now and we can always put it back in the future if desired.